hog.py

```diff
@@ -237,6 +237,9 @@ def run_experiments():
 
 # Strategies
 
+# BASELINE_NUM_ROLLS = 5
```

Comment:
Remove these comments as they would confuse someone ever reading this for the first time, since these global variables are already defined on lines 124 & 125. Additionally, their values could change and these comments would still show their old values, which violates the DRY principle.